omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/RestTestConstants1_10.java

```diff
@@ -26,4 +26,18 @@
 	public final static String COUGH_SYRUP_UUID = "0cbe2ed3-cd5f-4f46-9459-26127c9265ab";
 	
 	public final static String ASPIRIN_UUID = "15f83cd6-64e9-4e06-a5f9-364d3b14a43d";
+	
+	public final static String DRUG_REFERENCE_MAP_TEST_DATASET= "DrugReferenceMap1_10Test_Dataset.xml";
+	
+	public final static String DRUG_REFERENCE_DRUG_UUID= "8e2323fa-0fa0-461f-9b59-6765997d849d";
```

Comment:
Can you put space around the equal signs?

Reply:
i corrected it